src/main/java/edu/cornell/kfs/ksr/document/validation/impl/SecurityGroupRule.java

```diff
@@ -32,18 +33,15 @@ public boolean processCustomAddCollectionLineBusinessRules(MaintenanceDocument d
         if (line instanceof SecurityGroupTab) {
             SecurityGroup securityGroup = (SecurityGroup) document.getDocumentDataObject();
             SecurityGroupTab tempTab = (SecurityGroupTab) line;
-            if (securityGroup.getSecurityGroupTabs() != null) {
-                success = validateSecurityGroupTabs(document);
-                if (success) {
-                    for (SecurityGroupTab tab : securityGroup.getSecurityGroupTabs()) {
-                        if (tab.getTabOrder().equals(tempTab.getTabOrder())) {
-                            GlobalVariables.getMessageMap().putError(KSRPropertyConstants.SECURITY_GROUP_TAB_ORDER, KSRKeyConstants.ERROR_SECURITY_GROUP_TAB_ORDER_UNIQUE);
-                            success = false;
-                        }
-                        if (tab.getTabName().equals(tempTab.getTabName())) {
-                            GlobalVariables.getMessageMap().putError(KSRPropertyConstants.SECURITY_GROUP_TAB_NAME, KSRKeyConstants.ERROR_SECURITY_GROUP_TAB_NAME_UNIQUE);
-                            success = false;
-                        }
+            if (CollectionUtils.isNotEmpty(securityGroup.getSecurityGroupTabs())) {
+                for (SecurityGroupTab tab : securityGroup.getSecurityGroupTabs()) {
+                    if (tab.getTabOrder() != null && tab.getTabOrder().equals(tempTab.getTabOrder())) {
+                        GlobalVariables.getMessageMap().putError(KSRPropertyConstants.SECURITY_GROUP_TAB_ORDER, KSRKeyConstants.ERROR_SECURITY_GROUP_TAB_ORDER_UNIQUE);
+                        success = false;
+                    }
+                    if (StringUtils.isNotBlank(tab.getTabName()) && StringUtils.equals(tab.getTabName(), tempTab.getTabName())) {
```

Comment:
Do we need to be concerned about case with this compare?

Reply:
Yes, I agree that it makes sense to convert the noted comparisons to be case-insensitive. I will work on the appropriate updates.

Reply:
Updated to be case-insensitive.